packages/atomic/src/components/search/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -42,6 +49,25 @@ export class AtomicResultImage implements InitializableComponent {
     return Array.isArray(value) ? value[0] : value;
   }
 
+  private get altText() {
```

Comment:
As it is, this will return `any` and I think it should always return `string`.

If the specified field contains anything but a string or a string[] we should return the default alt (and maybe show a warning, although that warning would likely be shown for every result in the page...)